docs/pages/desktop-access/getting-started.mdx

```diff
@@ -145,7 +147,26 @@ following for the Windows Desktop Service:
 
 1. (Optional) Add labels for the Windows Desktop Service to the configuration file.
 
-   You can attach labels to Windows computers by matching to their hostnames.
+   You can add labels to Windows computers using `label` section in host config. For example, the following
+   adds the `important: true` label:
```

Comment:
I wouldn't use `true` as an example here, just because labels is a `map[string]string` and `true` will want to unmarshal to a bool by default.

Reply:
I updated it to `datacenter: dc1`